src/app/components/Select/Select.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { ChangeEvent } from 'react';
+
+type Props = {
+  name: string;
+  label: string;
+  defaultValue: string;
+  options: string[];
+  onChange(e: ChangeEvent): void;
+};
+
+function Select({ name, label, defaultValue, options, onChange }: Props) {
+  return (
+    <label htmlFor={name}>
+      {label}
+      <select name={name} onChange={onChange} defaultValue="">
+        <option hidden disabled value={defaultValue}>
+          select option
+        </option>
+        {options.map((option) => (
+          <option value={option} key={crypto.randomUUID()}>
```

Comment:
`key={crypto.randomUUID()}` 
This doesn’t make any sense to generate keys because on each render they will provide a new value. You can create a const with options[] and generate ids there, then pass it here so the ids will stay the same on every render. Or you can pass index since the options are immutable